cypress/integration/T2.js

```diff
@@ -0,0 +1,118 @@
+import LoginPage from '../objects/LoginPage';
+import TimeLoggin from '../objects/TimeLoggin';
+import Common from '../objects/Common';
+const common = new Common();
+const timeLoggin = new TimeLoggin();
+const loginPage = new LoginPage();
+
+describe('Sourcebooks login', function() {
+    it('Should display validation for empty user after attempted loggin', function () {
+        
+        loginPage.visit();
+        loginPage.getUserValidationIndicator().should('not.visible')
+        loginPage.getSubmitButton().click();
+        loginPage.getUserValidationIndicator().should('be.visible')
+    })
+
+    it('Should be able to login with user Jaunius_Mykolaitis to role Admin', function () {
+
+        loginPage.visit();
+        loginPage.getUserDropdown().click({force:true});
+        loginPage.getSpecificUserFromDropDown("Jaunius Mykolaitis").click();
+        loginPage.getRoleDropdown().click({force:true});
+        loginPage.getSpecificUserRoleFromDropDown("Admin").click();
+        loginPage.getSubmitButton().click();
+
+        timeLoggin.urlIncludeTimeLogging().should('include', '/time-logging');
+        timeLoggin.getPageTitle().contains('Timesheets')
+        timeLoggin.getCalendar().should('be.visible')
+        timeLoggin.getTimeTile().should('be.visible')
+        common.getUserName().contains('Jaunius Mykolaitis');
+        timeLoggin.getMainNavWithSpecificLength().should('have.length', 6);
+
+        timeLoggin.getMainNavActiveLinkColor();
```

Comment:
you are getting it but not doing anything with it, if you perform validation inside a function it would be nice to mention that in function name